nucypher/cli/commands/stake.py

```diff
@@ -570,7 +577,10 @@ def increase(general_config, transacting_staker_options, config_file, force, val
     #
 
     if not value:
-        token_balance = STAKEHOLDER.token_balance
+        if not only_lock:
```

Comment:
This conditional can be combined with the one above using `and`.

Reply:
not sure how to do this, or you mean do separate check before `if not value and not only_lock:`?

Reply:
Yes - unless there is the need for both levels, perhaps I am mis-reading the diff.

Reply:
not sure about that, `if not value:` has 3 inner `if` conditions, which represents interactive way to choose `value`, so extracting only one of them looks not clear